fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -1098,52 +1193,56 @@ protected DateTime getCurrentDate(DateTime dateTime, TimeGrain timeGrain) {
     }
 
     /**
+     * Extract valid sort direction.
+     *
+     * @param columnWithDirection  Column and its sorting direction
```

Comment:
I better understand things now. This isn't a String, but a `List<String>` which is expected to have 2 elements, a column name and a sort direction. It's a little odd, but I realize now that it's part of parsing.